command.go

```diff
@@ -137,7 +137,19 @@ func (c Command) Run(ctx *Context) (err error) {
 				flagArgs = args[firstFlagIndex:]
 			}
 
-			err = set.Parse(append(flagArgs, regularArgs...))
+			// separate combined flags
+			var flagArgsSeparated []string
+			for _, flagArg := range flagArgs {
+				if strings.HasPrefix(flagArg, "-") && strings.HasPrefix(flagArg, "--") == false && len(flagArg) >2 {
```

Comment:
Why `len(flagArg) > 2`?

Reply:
Probably should be just one I guess.

Reply:
Actually I think this is correct.  Since the flagArg starts with - followed by at least one char, then he needs to make sure their are two or more chars after the "-"

-t is valid and should be just handled

-ti 
Should be treated as -t and -i
-tip should be treated as -t -i -p

Reply:
Yeah, that's right.

Reply:
👍